gammapy/catalog/__init__.py

```diff
@@ -5,6 +5,8 @@
 from .fermi import *
 from .gammacat import *
 from .hawc import *
+from .hawc import SourceCatalogObject2HWC
```

Comment:
Here you don' need to explicitly import the `SourceCatalogObject`. It must be properly defined in gammapy.catalog.hawc.py. See below